App/src/pages/dossier/DossierReview.tsx

```diff
@@ -226,10 +244,11 @@ export default function DossierReview() {
                                 height="40px"
                                 //isLoading={loading}
                                 loadingText="Deleting"
-                                // onClick={() => {
-                                //     rejectDossier(dossier);
-                                //     // onClose();
-                                // }}
+                                onClick={() => {
+                                    window.location.reload();
```

Comment:
After forward/return/delete, it makes more sense in my opinion to redirect the user to the dossiers requiring review page, rather than just reloading the current page.

Reply:
Fixed.